packages/devtools-launchpad/webpack.config.devtools.js

```diff
@@ -4,20 +4,12 @@ const webpack = require("webpack");
 
 const { DefinePlugin } = webpack;
 
-const ignoreRegexes = [
-  /(chrome-remote-debugging-protocol)/
-];
```

Comment:
we should really switch this to `chrome-remote-interface`, but in reality i'm willing to take the bundle hit for the time being and remove it later when we improve the process for dropping unused modules.